tests/integration/metrics_test.go

```diff
@@ -211,3 +213,46 @@ func TestMetricsHealth(t *testing.T) {
 		t.Fatalf("expected '0' from etcd_server_health_failures, got %q", hv)
 	}
 }
+
+func TestMetricsRangeDurationSeconds(t *testing.T) {
+	integration.BeforeTest(t)
+	clus := integration.NewCluster(t, &integration.ClusterConfig{Size: 1})
+	defer clus.Terminate(t)
+
+	client := clus.RandClient()
+
+	_, err := client.Put(context.Background(), "my-namespace/foobar", "data")
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = client.Put(context.Background(), "my-namespace/foobar1", "data")
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = client.Put(context.Background(), "namespace/foobar1", "data")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Range over all keys.
+	_, err = client.Get(context.Background(), "", clientv3.WithFromKey())
+	if err != nil {
+		t.Fatal(err)
+	}
+	rangeDurationSeconds, err := clus.Members[0].Metric("etcd_server_range_duration_seconds")
+	if err != nil {
+		t.Fatal(err)
+	}
+	fmt.Println(rangeDurationSeconds)
```

Comment:
Please use `t.Logf` instead of `fmt.Println`

Reply:
I didn't mean to leave that print statement in there. I'll remove it.